magenta/musescore/magenta.qml

```diff
@@ -0,0 +1,109 @@
+import QtQuick 2.0
+import MuseScore 1.0
+
+MuseScore {
+      menuPath: "Plugins.Magenta"
+
+      function createNoteSequence() {
+        var noteSequence = {
+          timeSignatures: [],
+          keySignatures: [],
+          tempos: [],
+          notes: [],
+          totalTime: curScore.duration,
```

Comment:
I'll add a wrapper proto to include the user-selected portion of the score that the model should fill in.